src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -186,10 +180,9 @@ export class NativeStakingMapper {
       }),
     ]);
     this.validateDeployment(deployment);
-    const publicKeys =
-      this.kilnNativeStakingHelper.getPublicKeysFromDataDecoded(
-        args.dataDecoded,
-      );
+    const publicKeys = this.kilnNativeStakingHelper.splitPublicKeys(
+      this.kilnDecoder.decodeValidatorsExit(args.data)!.parameters[0].value,
```

Comment:
There is a TODO added to the `KilnDecoder` to simplify this after deprecating the `/transaction-confirmations` endpoint.